tap-snapshots/test-unit-ts.js-TAP.test.js

```diff
@@ -0,0 +1,91 @@
+/* IMPORTANT
+ * This snapshot file is auto-generated, but designed for humans.
+ * It should be checked into source control and tracked carefully.
+ * Re-generate by setting TAP_SNAPSHOT=1 and running tests.
+ * Make sure to inspect the output below.  Do not ignore changes!
+ */
+'use strict'
+exports[`test/unit/ts.js TAP TS Unit Tests > Pre-Processor processes input correctly 1`] = `
+Object {
+  "extension": ".mjs",
+  "source": "export const placeholder = 42;\\nexport const placeholder2 = [42, 42, 42];\\n",
+}
+`
+
+exports[`test/unit/ts.js TAP TS Unit Tests > Pre-Processor processes input correctly 2`] = `
+Object {
+  "extension": ".mjs",
+  "source": "\\"use strict\\";\\nObject.defineProperty(exports, \\"__esModule\\", { value: true });\\nexports.placeholder2 = exports.placeholder = void 0;\\nexports.placeholder = 42;\\nexports.placeholder2 = [42, 42, 42];\\n",
+}
+`
+
+exports[`test/unit/ts.js TAP TS Unit Tests > Pre-Processor processes input correctly 3`] = `
+Object {
+  "extension": ".mjs",
+  "source": "export const placeholder = 42;\\nexport const placeholder2 = [42, 42, 42];\\nconsole.log(placeholder);\\nconsole.log(placeholder2);\\n",
+}
+`
+
+exports[`test/unit/ts.js TAP TS Unit Tests > [TS Build - Bare Imports] Correct output source code 1`] = `
+import * as ts from 'typescript';
+console.log(\`Script Target: \${ts.ScriptTarget.ES2020}\`);
+
+console.log('This line was added by a post processor!!');
+`
+
+exports[`test/unit/ts.js TAP TS Unit Tests > [TS Build - Bare Imports] Correct output tree file names 1`] = `
+Array [
+  "file:///{fs}/presm/dist/test/fixtures/ts-build-bare/main.mjs",
+]
+`
+
+exports[`test/unit/ts.js TAP TS Unit Tests > [TS Build - Basic] Correct output source code 1`] = `
+const myBear = {
+    teeth: 10,
+    type: 'grizzly',
+    children: [{ teeth: 11, type: 'grizzly', children: [] }],
+};
+console.log(myBear);
+
+console.log('This line was added by a post processor!!');
```

Comment:
nit: It feels a bit weird to have this line in here. Maybe better to remove the post processor until there's a way to have more control over when it's added?

Reply:
Makes sense, removing now

Reply:
Removed this now